.nycrc

```diff
@@ -7,8 +7,8 @@
         "lib/**/*.js"
     ],
     "lines": 97,
-    "statements": 91,
+    "statements": 97,
     "functions": 100,
-    "branches": 89,
+    "branches": 91,
```

Comment:
Can we return old values and add more tests?

Reply:
/cc @Timer

Reply:
Just to clarify, you want me to reduce target test coverage and add then add more tests?

I increased the value(s) here with the tests I have already added -- I did not decrease them.